### DISCUSSION TITLE
What protocols are supported for a contract negotiation?

### DISCUSSION THREAD

**Discussion by dilip-dhankecha** [eclipse-edc/Connector#1094] [Q&A] [answered]
I see all the samples and in the contract negotiation, we need to specify the protocol and in Protocols.java there is only one constant which is ids-multipart.

What does that mean and what other protocols are supported?

**Comment by juliapampus** [comment, marked as answer]
Currently, the contract negotiation only supports IDS multipart. This is why there is one constant, that may be extended to include other protocols (specifications coming from e.g. IDS and GAIA-X) in the future. Nevertheless, the actual data transfer supports other protocols, as e.g. http/s. Those implementations are located at `extensions`.
